.github/workflows/main.yml

```diff
@@ -13,7 +13,7 @@ jobs:
         uses: erlef/setup-elixir@v1
         with:
           otp-version: 23.1.1
-          elixir-version: 1.11.0
+          elixir-version: 1.12.0
```

Comment:
```suggestion
          elixir-version: 1.12
```

Reply:
We should depend on a single version of Elixir to prevent potential future breakage.